packages/bot-web-ui/src/stores/server-side-bot-store.ts

```diff
@@ -475,7 +479,7 @@ export default class ServerBotStore {
                         amount: data.stake,
                         basis: 'stake',
                         contract_type: data.type,
-                        currency: this.root_store?.app?.core?.client?.currency || 'USD',
+                        currency: this.client_store?.currency || 'USD',
```

Comment:
```suggestion
                        currency: this.client_store?.currency ?? 'USD',
```

@sandeep-deriv Bhai minor nitpick.